scripts/video.js

```diff
@@ -154,6 +155,29 @@ H5P.Video = (function ($, ContentCopyrights, MediaCopyright, handlers) {
       return handlerName;
     };
 
+    /**
+    * @public
+    * Get current state for resetting it later.
+    *
+    * @returns {object} Current state.
+    */
+    self.getCurrentState = function () {
+      return {
+        startAt: self.getCurrentTime() || parameters.startAt
```

Comment:
Would it make more sense for the state object to contain a `currentTime` property instead of `startAt`? To me it seems that `startAt` shouldn't actually ever change. We are instead interested in the current time.